cmake/kokkos_arch.cmake

```diff
@@ -28,6 +28,7 @@ KOKKOS_CHECK_DEPRECATED_OPTIONS(
 #-------------------------------------------------------------------------------
 SET(KOKKOS_ARCH_LIST)
 
+include(CheckCXXCompilerFlag)
```

Comment:
I don't like where you placed that include but it is probably not worth re-running the CI